packages/patternfly-4/react-core/src/components/Select/Select.d.ts

```diff
@@ -0,0 +1,15 @@
+import { HTMLProps, FormEvent } from 'react';
+
+export interface SelectProps extends HTMLProps<HTMLOptionElement> {
+  isExpanded?: boolean;
+  onToggle(value: boolean): void;
+  placeholderText?: string;
+  selectOptions?: ReactNode[];
```

Comment:
We should remove selectOptions and just support children instead of having two ways of doing the same thing.

Reply:
I don't think its bad to have alternative ways of declaring the objects. If we're set on only supporting one way, for SelectOption, should we only support children instead? That would match how DropdownItem behaves.

Reply:
Though with select it makes more sense to use the prop. I'll make the changes as stated for the time being.